docs/reference/announcements.md

```diff
@@ -115,10 +115,16 @@ The `CorrelationResult` record has been changed compared to the previous version
 
 An example of how to use the new `CorrelationResult` can be found in the [Connector SDK documentation](/components/connectors/custom-built-connectors/connector-sdk.md#inbound-connector-runtime-logic).
 
-### Separated Ingress deprecation warning
+### Camunda 8 Self-Managed
+
+#### Helm chart - Separated Ingress deprecation
 
 The separated Ingress Helm configuration for Camunda 8 Self-Managed has been deprecated in 8.6, and will be removed from the Helm chart in 8.7. Only the combined Ingress configuration is officially supported. See the [Ingress guide](/self-managed/setup/guides/ingress-setup.md) for more information on configuring a combined Ingress setup.
 
+#### Helm chart - Elasticsearch nodes number
+
+The default value of Elasticsearch deployment pods has changed from 2 to 3 with an affinity setting. This avoids scheduling Elasticsearch pods on the same Kubernetes worker.
```

Comment:
```suggestion
The default value of Elasticsearch deployment pods has changed from 2 to 3, and an affinity setting has been added to avoid scheduling Elasticsearch pods on the same Kubernetes worker.
```
I updated this for readability, but wasn't sure if it should be as I have it, or the above suggestion. If the suggestion is more correct, I can update in both places - otherwise, this is good to go from me.